sdk-base/src/main/java/com/mapbox/maps/extension/style/StyleContract.kt

```diff
@@ -10,6 +10,11 @@ interface StyleContract {
    * Define the interfaces for the Style Extension.
    */
   interface StyleExtension {
+
+    /**
+     * Total resource count including all sources, layers and images.
+     */
+    var resourceCount: Int
```

Comment:
```suggestion
    val resourceCount: Int
      get() = sources.size + images.size + layers.size + if (light != null) 1 else 0 + if (terrain != null) 1 else 0
```

Reply:
We are decreasing it in `MapboxMap` so will not work that way I guess.

Reply:
Done.